packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="x-related-prompt x-bg-neutral-10 x-p-24 x-flex x-flex-col x-gap-16"
```

Comment:
I'm not sure about the styling. Usually we don't add those styles to the x-component and we leave it to the consumer to style it as desired.